packages/java/tests/pom.xml

```diff
@@ -15,7 +15,7 @@
     <packaging>pom</packaging>
 
     <properties>
-        <testbench.element.version>24.4-SNAPSHOT</testbench.element.version>
+        <testbench.element.version>24.5-SNAPSHOT</testbench.element.version>
```

Comment:
Could we postpone upgrading `testbench.element` to the near future when it's released? Now, it is breaking the build.